Jenkinsfile

```diff
@@ -195,15 +195,19 @@ pipeline {
         stage("provision test") {
             when {
                 expression {
-                    return pullRequestContainsLabels("test-provision,test-provision-aws,test-provision-gce,test-provision-docker,test-provision-k8s-local-kind-aws,test-provision-k8s-eks,test-provision-azure")
+                    def labels = [
+                        "test-provision", "test-provision-aws", "test-provision-gce", "test-provision-aws-reuse", "test-provision-gce-reuse",
+                        "test-provision-docker", "test-provision-k8s-local-kind-aws", "test-provision-k8s-eks", "test-provision-azure"
```

Comment:
reuse on azure?

Reply:
Missed azure, and intentionally skipped k8s one as when this change was implemented, there was an issue with k8s reuse (closed already).
Will check new labels directly in Jenkins for now (update/replay existing builds) and then will update this pipeline, if all is ok

Reply:
For azure and k8s-eks the provision test itself was successful, but with reuse failed.
For k8s-local-kind-aws the provision test itself failed.

I suggest to raise a separate issue(s) to fix the corresponding backends, and enable reuse for them in PR provision pipeline? (this PR continues to grow and would become hard to maintain).
For now the provision test reuse is working against AWS and GCE.
What do you think? @fruch @vponomaryov

Reply:
I think that's o.k., I'm o.k. with reuse cluster broken for those functional tests.
please raise an issue, with the details, and we can move this along

Reply:
raised https://github.com/scylladb/scylla-cluster-tests/issues/8520

Reply:
> raised #8520

- It should have been 2 separate issues - 1 for azure and 1 for K8S.
- Those 2 issues must contain some more info about failure, at least, link to a test run CI build, which is marked `stay alive`
- `azure` and `k8s` reuse labels be added, it doesn't break anything. Moreover, it will be much more useful to test further fixes to azure and k8s cases.